strongbox-web-core/pom.xml

```
@@ -103,36 +131,97 @@
+                            <systemPropertyVariables>
+                                <strongbox.basedir>${project.build.directory}</strongbox.basedir>
+                                <logging.config>${project.build.directory}/strongbox/etc/logback-spring.xml</logging.config>
+                                <logging.debug>true</logging.debug>
+                            </systemPropertyVariables>
+                        </configuration>
 
+                        <executions>
+                            <execution>
+                                <id>spring-boot-repackage</id>
+                                <goals>
+                                    <goal>repackage</goal>
+                                </goals>
+                                <phase>package</phase>
+
+                                <configuration>
+                                    <classifier>spring-boot</classifier>
+                                    <mainClass>org.carlspring.strongbox.app.StrongboxSpringBootApplication</mainClass>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>java-11-build</id>
+            <activation>
+                <property>
+                    <name>jdk.version</name>
+                    <value>11</value>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.springframework.boot</groupId>
+                        <artifactId>spring-boot-maven-plugin</artifactId>
                         <configuration>
-                            <classifier>spring-boot</classifier>
                             <mainClass>org.carlspring.strongbox.app.StrongboxSpringBootApplication</mainClass>
+                            <systemPropertyVariables>
+                                <strongbox.basedir>${project.build.directory}</strongbox.basedir>
+                                <logging.config>${project.build.directory}/strongbox/etc/logback-spring.xml</logging.config>
+                                <logging.debug>true</logging.debug>
+                            </systemPropertyVariables>
+
+                            <jvmArguments>
+                                --add-opens java.base/java.lang.module=ALL-UNNAMED --add-opens java.base/jdk.internal.loader=ALL-UNNAMED --add-opens java.base/jdk.internal.ref=ALL-UNNAMED --add-opens java.base/jdk.internal.reflect=ALL-UNNAMED --add-opens java.base/jdk.internal.math=ALL-UNNAMED --add-opens java.base/jdk.internal.module=ALL-UNNAMED --add-opens java.base/jdk.internal.util.jar=ALL-UNNAMED --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED
+                            </jvmArguments>
```

Does this still work, if these are on separate lines like this:
```suggestion
                            <jvmArguments>
                                --add-opens java.base/java.lang.module=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.loader=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.ref=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.reflect=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.math=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.module=ALL-UNNAMED
                                --add-opens java.base/jdk.internal.util.jar=ALL-UNNAMED
                                --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED
                            </jvmArguments>
```